tests/virtualization/universal/hotplugging.pm

```diff
@@ -136,14 +136,16 @@ sub test_add_vcpu {
     }
     return if (is_xen_host && $guest =~ m/hvm/i);    # not supported on HVM guest
 
+    if ($sles_running_version eq '15' && $sles_running_sp eq '4' && is_xen_host && is_fv_guest($guest)) {
+        record_info 'Skip vCPU hotplugging', 'bsc#1188898 vCPU hotplugging does no work on SLE FV guest on SLE15-SP4 host';
```

Comment:
Why not record_soft_failure ?

Reply:
yup, update to use with record_soft_failure 
[
The latest validation run](https://openqa.suse.de/tests/7030710#)